lib/src/main/java/io/ably/lib/push/Storage.java

```diff
@@ -0,0 +1,16 @@
+package io.ably.lib.push;
+
+import java.lang.reflect.Field;
+
+public interface Storage {
+
+    void putString(String key, String value);
```

Comment:
Also this is out of the context, but can we add small documentation at the top of the interface, as to what it does, so
developers using this interface will know what to do with it, like in case of `getString` it's mandatory to return `defaultValue` if value we are looking for a particular `key` is not found

Reply:
Not sure why this one is closed

Reply:
Can we quickly add a small doc above each interface method, one liner should also be enough, same goes for interface name. When Javadoc API documentation is generated, it will help there too : )

Reply:
Done